s2i-play/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rhel7.2
+
+MAINTAINER Andrew Block <ablock@redhat.com>
+
+LABEL io.k8s.description="Platform for building Play Framework Applications" \
+    io.k8s.display-name="Play Framework" \
+    io.openshift.expose-services="8080:http" \
+    io.openshift.tags="builder,webserver,html,play" \
+    io.openshift.s2i.scripts-url="image:///usr/libexec/s2i" \
+    io.openshift.s2i.destination="/opt/app-root"
```

Comment:
I know origin stuff doesn't do this, but should we include a label with our source repo?